docs/cloud/rahti/tutorials/configuration.md

```diff
@@ -81,4 +81,34 @@ data:
 metadata:
   name: webhooksecret
   namespace: mynamespace     # set this to your project namespace
-```
\ No newline at end of file
+```
+
+### Edit a secret
+
+The process to edit a secret is not trivial. The idea is to retrieve the secret JSON definition, decode it, edit it, and then encode it back and replace it.
+
+* First you need to retrieve the different files/secrets inside the secret (the examples use jq to process the JSON files, but it can be done without it):
+
+```sh
+oc get secrets $SECRET_NAME -o json | jq ' .data | keys '
+```
+
+* Then choose one of the options and get the file/secret itself:
+
+```sh
+oc get secrets $SECRET_NAME -o json >secret.json
+jq '.data.$KEY_NAME ' secret.json | base64 -d >$KEY_NAME.file
+```
+
+* Edit the file with any editor.
+
+* Encode the new file and replace the previous value in the JSON file:
+
+```sh
+B64=$(base64 $KEY_NAME.file -w0)
+jq " .data.$KEY_NAME = \"$B64\" " secret.json
+oc replace -f secret.json
+```
+
+As you can the process can be a bit ofuscated.
```

Comment:
+see
+obfuscated

Reply:
Fixed, thank you

Reply:
The "+see" was probably a bit ambiguous so

s/As you can/As you can see/

Reply:
:sweat: 
fixed